ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/title/v1/title/_cq_design_dialog/.content.xml

```diff
@@ -0,0 +1,154 @@
+                                                name="./allowedTypes"
+                                                text="H5"
+                                                value="h5"/>
+                                            <h6
+                                                jcr:primaryType="nt:unstructured"
+                                                sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
+                                                checked="true"
+                                                name="./allowedTypes"
+                                                text="H6"
+                                                value="h6"/>
+                                        </items>
+                                    </field>
+                                    <type
+                                        granite:class="core-title-size-default"
+                                        jcr:primaryType="nt:unstructured"
+                                        sling:resourceType="granite/ui/components/coral/foundation/form/select"
+                                        fieldLabel="Default Type / Size"
+                                        name="./type">
+                                        <items jcr:primaryType="nt:unstructured">
+                                            <h1
+                                                jcr:primaryType="nt:unstructured"
+                                                text="H1"
+                                                value="h1"/>
+                                            <h2
+                                                jcr:primaryType="nt:unstructured"
+                                                text="H2"
+                                                value="h2"/>
+                                            <h3
+                                                jcr:primaryType="nt:unstructured"
+                                                text="H3"
+                                                value="h3"/>
+                                            <h4
+                                                jcr:primaryType="nt:unstructured"
+                                                text="H4"
+                                                value="h4"/>
+                                            <h5
+                                                jcr:primaryType="nt:unstructured"
+                                                text="H5"
+                                                value="h5"/>
+                                            <h6
+                                                jcr:primaryType="nt:unstructured"
+                                                text="H6"
+                                                value="h6"/>
+                                        </items>
+                                    </type>
+                                    <disableLink
```

Comment:
lets remove disableLink from design dialog too fi we are not supporting links

Reply:
I made it hidden, as we are using the resourceSuperType as wcm title (v3), so even after removing here, it would be shown in the edit dialog. Or shall I remove the resourceSuperType, until we support links?

Reply:
Once we support links, we can remove a lot of redundant code. And extend from the Title properly. Otherwise, other than styling I don't think there is much dependency left from wcm now.